plugins/scaffolder-backend/api-report.md

```diff
@@ -452,13 +460,19 @@ export interface RouterOptions {
   // (undocumented)
   additionalTemplateGlobals?: Record<string, TemplateGlobal_2>;
   // (undocumented)
+  auth?: AuthService;
+  // (undocumented)
   catalogClient: CatalogApi;
   concurrentTasksLimit?: number;
   // (undocumented)
   config: Config;
   // (undocumented)
   database: PluginDatabaseManager;
   // (undocumented)
+  discovery?: DiscoveryService;
+  // (undocumented)
+  httpAuth?: HttpAuthService;
```

Comment:
Realized that we're never really gonna need an `HttpService` for actions, since we've left the HTTP layer at that point